src/utils.h

```diff
@@ -69,6 +69,8 @@ void GetFilesInFolder(std::string folder,
 // Ensures that |path| ends in a slash.
 void EnsureEndsInSlash(std::string& path);
 
+std::string EscapeFileName(std::string path);
```

Comment:
Add comment that this converts a path like "foo/bar.cc" into "foo_bar.cc"

Reply:
Done